pkgs/applications/audio/ctrlr/default.nix

```diff
@@ -0,0 +1,73 @@
+{ stdenv, fetchFromGitHub, libGL, unzip, libX11, libbfd, zlib, udev, alsaLib, freetype
+, libXrandr, libXinerama, libXext, libXcomposite, libXcursor, libiberty, zenity, which
+, makeDesktopItem }:
+
+let
+  desktopItem = makeDesktopItem rec {
+    name = "Ctrlr";
+    exec = name;
+    desktopName = name;
+    genericName = "Ctrlr MIDI panel";
+    categories = "Audio;AudioVideo;";
+  };
+in stdenv.mkDerivation rec {
+  pname = "ctrlr";
+  version = "5.3.201";
+
+  src = fetchFromGitHub {
+    owner = "RomanKubiak";
+    repo = "ctrlr";
+    # Scripts/git-revision.sh
+    rev = "179e457a5280566df65645b49bebf6b424c3f4ca";
```

Comment:
Please track the 6.0 release except if there is a good reason not to.

Reply:
@SuperSandro2000 The reason is that the panel developers make their panels using 5.3.201 listed on ctrlr website
Otherwise I would like to use 6.0.35

Reply:
Maybe we want to overwrite the version for that package? Not sure here.